cfsdk/src/main/java/io/harness/cfsdk/cloud/model/Target.java

```diff
@@ -1,14 +1,32 @@
 package io.harness.cfsdk.cloud.model;
 
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import io.harness.cfsdk.utils.CfUtils;
+
 public class Target {
+
     private String name;
     private String identifier;
+    private boolean isPrivate; // If the target is private
```

Comment:
There is only 1 target in the system so not sure what this means if the target is private